lib/web_console/whiny_object.rb

```diff
@@ -0,0 +1,23 @@
+module WebConsole
+  class WhinyObject < SimpleDelegator
```

Comment:
@gsamokovarov Although I'm satisfied with the last approach, I'm looking forward to hear from you what is your preference.